code/server/test/unit-utils.js

```diff
@@ -302,6 +303,43 @@ exports.testGetPotentialStartEndPoints = function (itShould, hikeId, expectedGet
   });
 }
 
+exports.testGetPotentialHut = function (itShould, hikeId, expectedGetPotentialHutProperties) {
+  test(`Should ${itShould}`, async () => {
+    const res = await HikeManager.getPotentialHuts(hikeId);
+    expect(res).toHaveProperty(expectedGetPotentialHutProperties);
+    
```

Comment:
We could consider updating this test by having 2 points in the DB: one close to the hike and one further than the radius, and we could check that only one closest one is actually retrieved by the `getPotentialHuts`

Reply:
Updated the assert from property to the length